src/front-end/typescript/lib/pages/dashboard.tsx

```diff
@@ -305,13 +309,18 @@ const update: component_.page.Update<State, InnerMsg, Route> = updateValid(
         switch (msg.value.tag) {
           case "vendor": {
             const [cwuProposals, swuProposals, organizations] = msg.value.value;
-            const isQualified = organizations.reduce(
+            const isSWUQualified = organizations.reduce(
               (acc, o) => acc || doesOrganizationMeetSWUQualification(o),
               false as boolean
             );
+            const isTWUQualified = organizations.reduce(
+              (acc, o) => acc || doesOrganizationMeetTWUQualification(o),
+              false as boolean
+            );
```

Comment:
Opportunity for a helper here: `satisfies<A>(as: A[], fn: (a: A) => boolean): boolean`.

Reply:
I'm pretty sure this functionality could be replicated by Array.prototype.some, i.e. `organizations.some(o => doesOrganizationMeetTWUQualification(o))`. 

Not sure if that still warrants a helper function?

Reply:
Yup, `some` will do the trick!